VirtoCommerce.CatalogModule.Data/Services/CategoryServiceImpl.cs

```diff
@@ -100,6 +100,8 @@ public virtual void Delete(string[] categoryIds)
                 .Select(c => new GenericChangedEntry<Category>(c, EntryState.Deleted))
                 .ToList();
 
+            _commerceService.DeleteSeoForObjects(categories);
```

Comment:
Decided to not add new method to this interface to avoid possible compatibility problems with existing custom implementations, so please use existing `DeleteSeoForObject`.